scss/style.scss

```diff
@@ -0,0 +1,1241 @@
+body {
+  font-family: system-ui, -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue", sans-serif;
+}
+
+.wpmdb {
+  .wrapper {
+    max-width: 1280px;
+    margin: 100px;
+  }
+
+  .html {
+    font-size: 10px;
+  }
```

Comment:
I can't find this class in the markup

Reply:
This is actually a mistake. Originally I was trying to set the html (not .html) to a font-size of 10px as a baseline for my rem sizing.  Ill remove.